packages/visual-editing/src/ui/telemetry/sendTelemetry.ts

```diff
@@ -0,0 +1,34 @@
+import {defineEvent} from '@sanity/telemetry'
+import type {VisualEditingNode} from '../../types'
+
+const events = {
+  'Visual Editing Drag Sequence Complete': defineEvent({
+    name: 'Visual Editing Drag Sequence Complete',
+    description: 'An array is successfully reordered using drag and drop.',
+    version: 1,
+  }),
+}
```

Comment:
@svirs is this event formatting OK?

Reply:
@georgedoescode I'd make the event name past-tense `Completed`

Reply:
Thanks!